client/lib/perfmon/README.md

```diff
@@ -22,3 +22,17 @@ localStorage.setItem('debug', 'calypso:perfmon')
 ```
 
 and then reloading the browser.
+
+Note that while the timer is started and stopped on every navigation, the
+observer is only enabled on pages where it's explicitly initiated (such as
+the post editor). In all other pages, the timer won't stop when the placeholders
+are removed.
```

Comment:
Just to clarify: when "starting a timer", we just store the current timestamp in a variable inside `lib/perfmon`. We don't allocate any timer object and there are no memory leaks when a timer "won't stop".

It would be different if we used something like `console.time( 'timer-id' )` and didn't call `console.timeEnd( 'timer-id' )`. Then there would be an internal record in the console implementation keyed by `timer-id` that's never freed.